src/main/scala/beam/agentsim/infrastructure/ScaleUpCharging.scala

```diff
@@ -29,42 +32,42 @@ import scala.util.Random
 
 trait ScaleUpCharging extends {
   this: ChargingNetworkManager =>
+
   import ScaleUpCharging._
 
   private lazy val rand: Random = new Random(beamConfig.matsim.modules.global.randomSeed)
-  private lazy val mersenne: MersenneTwister = new MersenneTwister(beamConfig.matsim.modules.global.randomSeed)
   private lazy val timeStepByHour = beamConfig.beam.agentsim.chargingNetworkManager.timeStepInSeconds / 3600.0
   private lazy val virtualParkingInquiries: TrieMap[Int, ParkingInquiry] = TrieMap()
-  private lazy val vehicleRequests = mutable.HashMap.empty[(Id[TAZ], ParkingActivityType), List[VehicleRequestInfo]]
+  private lazy val vehicleRequests = mutable.HashSet.empty[ChargingEvent]
+  private lazy val tazTreeMap = getBeamServices.beamScenario.tazTreeMap
+
+  private lazy val scaleUpFactor: Double =
+    if (!cnmConfig.scaleUp.enabled) 0.0
+    else if (cnmConfig.scaleUp.expansionFactor >= 1.0) cnmConfig.scaleUp.expansionFactor - 1.0
+    else throw new RuntimeException(s"scaleUp.expansionFactor == ${cnmConfig.scaleUp.expansionFactor} < 1.0")
```

Comment:
I think all these `lazy` vals can be replace with just vals.

Reply:
Okay